content/blog/2021-05-15-may-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,239 @@
+﻿---
+title: May '21 Heartbeat
+date: 2021-05-15
+description: |
+  Monthly updates are here! We've hit 30 team members! MLOps learning 
+  opportunities, tutorials with integrations, conference videos, Discord 
+  server growth, and more!
+descriptionLong: |
+  This month you will find:
+  🚀 info on our growing team 
```

Comment:
I think content on the team growth can be included after the other sections (in this list as well as in the details below) to highlight the other sections more.